.github/workflows/test-integrations.yml

```diff
@@ -546,7 +538,9 @@ jobs:
             -p=4 \
             -tags "${{ env.GOTAGS }}" \
             -timeout=30m \
-            -json ./... \
+            -json \
+            ./... \
+            --follow-log=false \
```

Comment:
Log is disabled since consul logs doesn't provide any info since there are multiple tests running in parallel.